.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "submodules/crocoddyl"]
 	path = submodules/crocoddyl
-	url = https://github.com/cmastalli/crocoddyl.git
+	url = https://github.com/loco-3d/crocoddyl.git
 	branch = topic/memmo-school
```

Comment:
This branch doesn't exist on the main repo. You could either choose master or devel, or put the v1.4.0 tag, or use a `topic/memmo-school` branch on the main repo.